config/checkstyle_input_checks.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <property name="charset" value="UTF-8"/>
+    <property name="haltOnException" value="false"/>
+
+    <property name="cacheFile" value="target/cache_resources"/>
+
+    <!-- Suppressions for resources -->
+    <module name="SuppressionFilter">
+        <property name="file"
+                  value="${project.basedir}/config/checkstyle_input_suppressions.xml"/>
+    </module>
+
+    <!-- Check to make sure check UT input files have config and trailing comments -->
+    <module name="RegexpMultiline">
```

Comment:
I thought about Multiline only to test format of comment, but it was my fault.
We do not need multiline at all, we do not use benefit of it here.
Please use https://checkstyle.org/config_regexp.html#Regexp